DOCS/encoding.rst

```diff
@@ -133,19 +133,19 @@ What works
 
 * Encoding at variable frame rate (default)
 * Encoding at constant frame rate using --vf=fps=RATE
-* 2-pass encoding (specify flags=+pass1 in the first pass's -ovcopts, specify
+* 2-pass encoding (specify flags=+pass1 in the first pass's --ovcopts, specify
   flags=+pass2 in the second pass)
 * Hardcoding subtitles using vobsub, ass or srt subtitle rendering (just
   configure mpv for the subtitles as usual)
-* Hardcoding any other mpv OSD (e.g. time codes, using -osdlevel 3 and -vf
-  expand=::::1)
+* Hardcoding any other mpv OSD (e.g. time codes, using --osdlevel=3 and
+  --vf=expand=::::1)
 * Encoding directly from a DVD, network stream, webcam, or any other source
   mpv supports
 * Using x264 presets/tunings/profiles (by using profile=, tune=, preset= in the
-  -ovcopts)
+  --ovcopts)
 * Deinterlacing/Inverse Telecine with any of mpv's filters for that
-* Audio file converting: mpv -o outfile.mp3 infile.flac -no-video -oac
-  libmp3lame -oacopts ab=320k
+* Audio file converting: mpv --o=outfile.mp3 infile.flac --no-video
+  --oac=libmp3lame --oacopts=ab=320k
```

Comment:
I'd probably replace this with `outfile.m4a` and `--oac=aac` because nobody should produce new mp3s anymore.